### PR TITLE
chore: replace fmt.Errorf with errors.New where possible

### PR DESCRIPTION
**Pull request by DmitriyMV**
This time use `eg` from `x/tools` repo tool to do this.

### CODE REVIEW

File: cmd/talosctl/pkg/talos/action/node.go [hunk 1]
```diff
@@ -268,7 +268,7 @@ func (a *nodeTracker) handleEvent(event client.Event) error {
 		})
 
 		if msg.GetTask() == "stopAllServices" {
-			return retry.ExpectedError(fmt.Errorf("stopAllServices task completed"))
```
Comment (smira) [inline cmd/talosctl/pkg/talos/action/node.go:271]: can use `retry.ExpectedErrorf`

File: internal/app/machined/pkg/controllers/network/operator_merge_test.go [hunk 2]
```diff
@@ -293,7 +294,7 @@ func (suite *OperatorMergeSuite) TestMergeFlapping() {
 
 						if *override.TypedSpec() != *r.TypedSpec() {
 							// using retry here, as it might not be reconciled immediately
-							return retry.ExpectedError(fmt.Errorf("not equal yet"))
```
Comment (smira) [inline internal/app/machined/pkg/controllers/network/operator_merge_test.go:296]: same here

File: internal/app/machined/pkg/controllers/network/route_merge_test.go [hunk 3]
```diff
@@ -201,7 +202,7 @@ func (suite *RouteMergeSuite) TestMerge() {
 						case "inet4/10.5.0.3//50":
 							if *cmdline.TypedSpec() != *r.TypedSpec() {
 								// using retry here, as it might not be reconciled immediately
-								return retry.ExpectedError(fmt.Errorf("not equal yet"))
```
Comment (smira) [inline internal/app/machined/pkg/controllers/network/route_merge_test.go:204]: and here

File: internal/app/machined/pkg/controllers/perf/perf_test.go [hunk 4]
```diff
@@ -107,7 +107,7 @@ func (suite *PerfSuite) TestReconcile() {
 				memSpec := mem.(*perfresource.Memory).TypedSpec()
 
 				if len(cpuSpec.CPU) == 0 || memSpec.MemTotal == 0 {
-					return retry.ExpectedError(fmt.Errorf("cpu spec does not contain any CPU or Total memory is zero"))
```
Comment (smira) [inline internal/app/machined/pkg/controllers/perf/perf_test.go:110]: and here

**Comment by smira** [comment]
more we could use `retry.ExpectedErrorf`

File: cmd/talosctl/pkg/talos/action/node.go (hunk 1 above)
Comment (DmitriyMV) [reply cmd/talosctl/pkg/talos/action/node.go:271]: Done here and below.

**Pull request merged**